index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="styles.css" rel="stylesheet"><title>My Portfolio</title>
+    </head>
+    <body>
+        <header>
+            <p class="logo">Femi</p>
+            <a class="togglebar" href="#"><img src="Images/Icon-Menu.png" alt="menu icon"></a>
```

Comment:
It's a good idea to use semantic elements on the page when ever possible. Please wrap the hamburger icon with a nav element.